Runtime/Behaviours/CinemachineStoryboard.cs

```diff
@@ -120,12 +139,40 @@ class CanvasInfo
             if (vcam != VirtualCamera || stage != CinemachineCore.Stage.Finalize)
                 return;
 
+            UpdateRenderCanvas();
+
             if (m_ShowImage)
                 state.AddCustomBlendable(new CameraState.CustomBlendable(this, 1));
             if (m_MuteCamera)
                 state.BlendHint |= CameraState.BlendHintValue.NoTransform | CameraState.BlendHintValue.NoLens;
         }
 
+        /// <summary>
+        /// Camera render modes supported by CinemachineStoryboard.
+        /// </summary>
+        public enum StoryboardRenderMode
+        {
+            ScreenSpaceOverlay,
+            ScreenSpaceCamera
```

Comment:
these should be defined in terms of RenderMode values - code elsewhere is making that assumption